external/importL10n/locales.mjs

```diff
@@ -71,41 +61,27 @@ function downloadLanguageFiles(root, langCode) {
 
   // Defines which files to download for each language.
   const files = ["viewer.ftl"];
-  let downloadsLeft = files.length;
 
   const outputDir = path.join(root, langCode);
   if (!fs.existsSync(outputDir)) {
     fs.mkdirSync(outputDir);
   }
 
-  return new Promise(function (resolve) {
-    // Download the necessary files for this language.
-    files.forEach(function (fileName) {
-      const outputPath = path.join(outputDir, fileName);
-      const url =
-        MOZ_CENTRAL_ROOT + langCode + MOZ_CENTRAL_PDFJS_DIR + fileName;
-
-      https.get(url, function (response) {
-        // Not all files exist for each language. Files without translations
-        // have been removed (https://bugzilla.mozilla.org/show_bug.cgi?id=1443175).
-        if (response.statusCode === 200) {
-          let content = "";
-          response.setEncoding("utf8");
-          response.on("data", function (chunk) {
-            content += chunk;
-          });
-          response.on("end", function () {
-            fs.writeFileSync(outputPath, normalizeText(content), "utf8");
-            if (--downloadsLeft === 0) {
-              resolve();
-            }
-          });
-        } else if (--downloadsLeft === 0) {
-          resolve();
-        }
-      });
-    });
-  });
+  // Download the necessary files for this language.
+  for (const fileName of files) {
```

Comment:
The old implementation was downloading files in parallel, while this one is downloading them sequentially.

I don't know how many files this function downloads so I don't know if it actually has any impact, but in case it does you can still do it in parallel while using async/await:

```js
return Promise.all(files.map(async fileName => {
  const outputPath = path.join(outputDir, fileName);
  const url = MOZ_CENTRAL_ROOT + langCode + MOZ_CENTRAL_PDFJS_DIR + fileName;

  const response = await fetch(url);
  if (!response.ok) {
    // Not all files exist for each language. Files without translations
    // have been removed (https://bugzilla.mozilla.org/show_bug.cgi?id=1443175).
    continue;
  }
  const content = await response.text();

  await fs.promises.writeFile(outputPath, normalizeText(content), "utf8"));
}));
```

Reply:
> I don't know how many files this function downloads so I don't know if it actually has any impact,

Currently, and for quite some time, it's only a single file per locale; see https://github.com/mozilla/pdf.js/blob/72b8b29147c3a19ccacc9db111df4925efcbe1bf/external/importL10n/locales.mjs#L72-L73

Besides, given how this code is used, absolute maximum speed isn't really a concern hence I'd prefer keeping this simple.